content/guides/references/migration-guide.md

```diff
@@ -207,6 +207,16 @@ alias to be the first button rather than the unexpanded button.
 cy.findByTestId('popover').findAllByRole('button').first().as('button')
 ```
 
+If you want to alias a static value, such that it is never re-queried, you will
+need Cypress 12.3.0 or later, which introduced the `type` option for `.as()` to
```

Comment:
Need to add link to changelog & to `.as()` command.

Reply:
Added.